File: BackEnd/Config/Mailer.js
const nodemailer = require('nodemailer');

const transporter = nodemailer.createTransport({
    service: 'gmail',
    auth: {
        user: '0323105925@ut-tijuana.edu.mx',
        pass: 'gheb ytwn qscb bwyb'
    }
});

const enviarCodigo = async (correoDestino, codigo) => {
    const mailOptions = {
        from: 'tucorreo@gmail.com',
        to: correoDestino,
        subject: 'CORREO DE VALIDACION',
        text: `${codigo}`
    };
    return transporter.sendMail(mailOptions);
};

module.exports = { enviarCodigo };

File: BackEnd/core/utils/mailer.py
import yagmail

EMAIL_USER = '0323105925@ut-tijuana.edu.mx'
EMAIL_PASS = 'gheb ytwn qscb bwyb'

def enviar_codigo(correo_destino, codigo):
    yag = yagmail.SMTP(EMAIL_USER, EMAIL_PASS)
    yag.send(
        to=correo_destino,
        subject='Código de verificación - Invoxa',
        contents=f'Tu código de recuperación es: {codigo}'
    )